app/helpers/orchestration/orchestrator.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+# Orchestrator: Methods for managing active playbook context
+module Orchestration::Orchestrator # rubocop:todo Metrics/ModuleLength
+  include Orchestration::Playbooks
+
+  available = Orchestration::Playbooks.constants.reject { |m| m == :Templates }
+  modules = available.map { |m| ('Orchestration::Playbooks::' + m.to_s).constantize }
+
+  playbooks = {}
+  modules.each { |m| playbooks[m::NAME] = m::PLAYBOOK }
+
+  PLAYBOOKS = playbooks.freeze
+
+  def workflow_configuration(playbook, workflow, option)
+    # For now, assume that if no workflow (ie it is nil, it is a general option)
+    base_configuration = workflow.nil? ? PLAYBOOKS.dig(playbook, :general, :base, option) : PLAYBOOKS.dig(playbook, :workflows, workflow, :base, option)
+    custom_options = workflow.nil? ? PLAYBOOKS.dig(playbook, :general, :custom_options,
+                                                   option) : PLAYBOOKS.dig(playbook, :workflows, workflow, :custom_options, option)
+
+    return if base_configuration.nil?
+    return base_configuration if custom_options.nil?
+
+    nested_configuration(base_configuration, custom_options)
+  end
+
+  # 'base': Return the whole base configuration. Do not follow nested configurations
+  # 'all': Return the base configuration options, but also follow the nested configurations
+  # 'subset': Select the given keys from the options, but also follow the nested configurations
+  # 'remove': Reject the given keys from the options, but also follow the nested configurations
+  # 'custom': Overwrite the whole configuration
+  def nested_configuration(base_configuration, playbook_configuration)
+    # Replace label first if available
+    unless base_configuration[:label].nil?
+      base_configuration[:label] =
+        playbook_configuration[:label].nil? ? base_configuration[:label] : playbook_configuration[:label]
+    end
```

Comment:
```suggestion
    base_configuration[:label] = playbook_configuration[:label] unless base_configuration[:label].nil? || playbook_configuration[:label].nil?
```

Reply:
@ChristineDuong

Reply:
This has been resolved
https://github.com/SaraAlert/SaraAlert/blob/559e5ae780bc9ea07e123e1e985c449a1ee71706/app/helpers/orchestration/orchestrator.rb#L36